nodepool/elements/nodepool-base/extra-data.d/99-copy-nodepool-scripts

```diff
@@ -33,7 +33,7 @@ fi
 set -eu
 set -o pipefail
 
-NODEPOOL_SCRIPTDIR=${NODEPOOL_SCRIPTDIR:-$TMP_MOUNT_PATH/opt/git/openstack-infra/project-config/nodepool/scripts}
+NODEPOOL_SCRIPTDIR=${NODEPOOL_SCRIPTDIR:-$TMP_MOUNT_PATH/opt/git/a10networks-ci/project-config/nodepool/scripts}
```

Comment:
Can we stash as much of these things in ENV variables where possible, ideally since these scripts live in the same place they would source the same set of vars from a base script or similar.